spec/lib/vmdb/settings_spec.rb

```diff
@@ -566,6 +566,47 @@
       settings = Vmdb::Settings.for_resource(MiqServer.new(:zone => server.zone))
       expect(settings.api.token_ttl).to eq "4.hour"
     end
+
+    context "on global region" do
+      let(:key)                 { "yuri" }
+      let(:value_region_remote) { "region_remote" }
+      let(:value_server_remote) { "server_remote" }
+
+      before do
+        Zone.seed
+        @region_global = MiqRegion.first
+        @region_remote = FactoryBot.create(:miq_region, :id => MiqRegion.id_in_region(1, @region_global.region + 1))
```

Comment:
I think all these calls to `id_in_region` would make sense if they just used `ApplicationRecord` rather than a specific model.

Using the model doesn't actually change the behavior, but it does make a reviewer think it might.

Reply:
right, i just copied it from somewhere